Copying.md

```diff
@@ -12,4 +12,9 @@ Modifications by members of the FreeSpace Source Code Project are
 released under whatever terms the individual authors choose, but the
 above notice continues to apply to all fs2_open code.
 
+All modifications made on or after November 1, 2020, shall be released under 
+the terms within [Unlicense](Unlicense.md). Modifications made before said date are released 
+under the previous terms with the exception of modifications made by members 
+of the FreeSpace Source Code Project that opt-in to the terms of Unlicense.
```

Comment:
Maybe link to the opt-in list or reference it like "The opt-in list may be found in..."